### PR TITLE
Update metrics

### LINKED ISSUE #1487: User Metrics: Questions
Is it possible for telemetry to answer the following questions:

How many games is a user managing through NMA?
How many mods does a user have installed? will need this per game. 
How many mods has the user downloaded?
How long does it take for a game to be managed? per game
What system specs are being used?
How many loadouts are being created per game?
How many people have undo turned on to turned off?
Can we track bugs or problems with the app?

Is there anything we can add that would tell us if there is a problem with the app?

- #1184

### PR DESCRIPTION
**Pull request by erri120**
Resolves #1487.

- [x] Users per Version
- [x] Managed Games Count
- [x] Mods Per Game (avg Loadout count)
- [x] Global Download Size

### CODE REVIEW

**Comment by Al12rs** [comment]
Is average mod count more useful than max? Should we also add Max?

**Review by Al12rs — APPROVED** [review]

File: src/NexusMods.App/TelemetryProvider.cs [hunk 1]
```diff
@@ -1,30 +1,55 @@
...
+            .ToObservableChangeSet()
+            .Sum(x => (double)x.Size.Value)
+            .SubscribeWithErrorLogging(totalDownloadSize => _downloadSize = Size.From((ulong)totalDownloadSize))
+            .DisposeWith(_disposable);
     }
 
     public void ConfigureMetrics(IMeterConfig meterConfig)
     {
+#if RELEASE
+        meterConfig.CreateActiveUsersCounter(BuildInfo.ApplicationConstants.Version);
```
Comment (Al12rs) [inline src/NexusMods.App/TelemetryProvider.cs:43]: Ugh, this is actually broken I think

Comment (Al12rs) [reply src/NexusMods.App/TelemetryProvider.cs:43]: @erri120 Is that correct?

Comment (erri120) [reply src/NexusMods.App/TelemetryProvider.cs:43]: Right, another case why preprocessor definitions suck.

Comment (Al12rs) [reply src/NexusMods.App/TelemetryProvider.cs:43]: Agreed, I will fix it in my pr since I need to fix conflicts in this file anyways

Comment (Al12rs) [reply src/NexusMods.App/TelemetryProvider.cs:43]: Pushed the fix as well

**Pull request merged**